### DISCUSSION TITLE
My game Crashing to the ending of quest !

### DISCUSSION THREAD

**Discussion by NoirroseLunalyn** [praydog/REFramework#874] [General Questions]
Help please My Monster hunter rise Sunbreak Freeze when the quest is done ! I use laptop to play this game I tried different version of Reframework spec is :  CPU AMD ryzen 5000series  GPU Radeon 6500rx ram:16gb windows 11 Please help I tried many different ways 
It's not crashing when Launching the game but to ending the quest the game's freeze Thank you for helping !

**Comment by NoirroseLunalyn** [comment]
![image](https://github.com/praydog/REFramework/assets/141849012/2a448a95-14b5-4d68-a1fb-ed194122ce79)

this is what's it looks like

**Comment by Darkblizzard21** [comment]
I also am encountering the issue. But only if the quest updates a crown size.
